wire/blockheader_test.go

```diff
@@ -216,14 +217,15 @@ func TestBlockHeaderWire(t *testing.T) {
 			continue
 		}
 
-		b, err := wire.TstBytesBlockHeader(test.in)
+		buf.Reset()
+		err = test.in.BtcEncode(&buf, pver)
```

Comment:
Yeah that would make sense.  I suspect `MsgEncode`/`MsgDecode` would make the most sense given it is part of the `Message` interface.